oauth/authorization.md

```diff
@@ -13,7 +13,7 @@ About 1.5 hour
 
 Here are links to lessons that should be completed before this lesson:
 
```

Comment:
@vegetabill should this be an elective if it has an elective prereq?

Reply:
OAuth is an elective

Reply:
@FrozenGirl-spd change this one too please and move the link to electives